db/migrate/20190923181351_create_external_registries.rb

```diff
@@ -0,0 +1,12 @@
+class CreateExternalRegistries < ActiveRecord::Migration
+  def change
+    create_table :external_registries do |t|
```

Comment:
I don't understand why these are in the DB, rather than just ruby classes not backed by the database

Reply:
Something like the models that use enums for associations, e.g. [HandlebarType](https://github.com/bikeindex/bike_index/blob/master/app/models/handlebar_type.rb)

Reply:
I think it might make more sense to follow the single table inheritance pattern (naming and file structure) for this, though making it a plain ruby object rather than an active record database backed thing makes more sense to me.

So the class would look like `ExternalRegistry::StopHeling`

This is sort of a nit, but I think it will fundamentally change how things look and work and make it less surprising.

Reply:
> I don't understand why these are in the DB, rather than just ruby classes not backed by the database

For the sake of data normalization — the alternative is to duplicate the registry name and url (and any other potentially associated attributes) across the `external_registry_bikes` table.

Reply:
Ahhh. But I think if we use the enums and classes we can achieve the same thing and don't need to use the database - also I think it better represents our data/architecture, since we will have a class for each client.

Reply:
>  But I think if we use the enums and classes we can achieve the same thing and don't need to use the database

Sure, but that would be denormalized data (still stored in the database), with no clear rationale for that denormalization. It also elides the associated data (should we have enums for external registry URLs and external registry countries too?)

You're confusing the `external_registries` table with the coupling of that table with client api classes. I see no reason to do away with the former. If we don't see a need for location-based selection of client classes, we can do away with the latter (which draws some inspiration from the way twitter accounts are currently handled—I'm surprised it's as illegible as it appears to be).